### DISCUSSION TITLE
`must NOT have additional properties` after upgrading mia-platform

### DISCUSSION THREAD

**Discussion by davidgarra** [mia-platform/community#296] [Q&A]
### Which section?

Console

### Which is the version?

0.4.1

### Description

Hi everyone, 
After updating mia-platform from version 0.4.0 to 0.4.1, I encountered an error on the CRUD-Service. Here are the details:

```
[1689923584288] WARN (8 on crud-service-56945b4986-nfhlq): collection using custom fields configuration - which has been deprecated
    collectionName: "active_codes"
[1689923584289] ERROR (8 on crud-service-56945b4986-nfhlq):
    0: {
      "instancePath": "",
      "schemaPath": "#/additionalProperties",
      "keyword": "additionalProperties",
      "params": {
        "additionalProperty": "tags"
      },
      "message": "must NOT have additional properties"
    }
Error: Invalid collection definition: active_codes
    at loadModels (/home/node/app/index.js:104:13)
    at Plugin.exec (/home/node/app/node_modules/avvio/plugin.js:132:19)
    at Boot.loadPlugin (/home/node/app/node_modules/avvio/plugin.js:274:10)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
```

I am having trouble understanding how to resolve this issue. Could anyone provide any suggestions or guidance on what changes I need to make? Thank you.

**Comment by danibix95** [comment]
Hi David,

could you please provides us more details regarding the CRUD Service, such as service version?
In case it is not a privacy problem, would it be possible also to share the collection definition of `active_codes` to provide more insights on the raised error?

In addition, when you mention Console version `0.4.1`, do you mean the latest one released, which is `v11.4.1`?

**Reply by davidgarra** [reply]
Yeah, I'm sorry, the version I am referring to is  `v11.4.1`. 
I was able to resolve the issue by upgrading the CRUD-service from version `6.7.0` to `6.8.0`. However, I couldn't find any information about breaking changes in the [Mia-Platform Changelog](https://docs.mia-platform.eu/docs/release-notes/v11.4.1) for  `v11.4.1`...

**Reply by danibix95** [reply]
OK, thank you for sharing more details.

Actually, we noticed we accidentally introduced a small breaking change on the generation of collection definition tags, which concerns exactly the error you encountered. We are already working on an hotfix to avoid further issues on this matter.

Your timely feedback is really appreciated!

**Comment by davidebianchi** [comment]
Hi @davidgarra, Friday we release an hotfix to address this specific issue: https://docs.mia-platform.eu/docs/release-notes/v11.4.2
Now, crud-service will also work with version below the 6.8.0. Thank you for the feedback!
